ossec_security/assets/dashboards/ossec_syslog.json

```diff
@@ -0,0 +1,1786 @@
+                            "indexes": [],
+                            "storage": "hot"
+                        },
+                        "columns": [
+                            {
+                                "field": "status_line",
+                                "width": "auto"
+                            },
+                            {
+                                "field": "timestamp",
+                                "width": "auto"
+                            },
+                            {
+                                "field": "@usr.name",
+                                "width": "auto"
+                            },
+                            {
+                                "field": "@usr.id",
+                                "width": "auto"
+                            },
+                            {
+                                "field": "description",
+                                "width": "auto"
+                            },
+                            {
+                                "field": "log",
+                                "width": "auto"
+                            }
+                        ]
+                    }
+                ],
+                "type": "list_stream"
+            },
+            "layout": {
+                "x": 0,
+                "y": 19,
+                "width": 12,
+                "height": 4
+            }
+        },
+        {
+            "id": 4287032187302792,
+            "definition": {
+                "title": "Top 10 Authentication Failed Events By Source IP",
+                "title_size": "16",
+                "title_align": "left",
+                "type": "toplist",
+                "requests": [
+                    {
+                        "queries": [
+                            {
+                                "data_source": "logs",
+                                "name": "query1",
+                                "indexes": [
+                                    "*"
+                                ],
+                                "compute": {
+                                    "aggregation": "count"
+                                },
+                                "group_by": [
+                                    {
+                                        "facet": "@network.client.ip",
+                                        "limit": 10,
+                                        "sort": {
+                                            "order": "desc",
+                                            "aggregation": "count"
+                                        }
+                                    }
+                                ],
+                                "search": {
+                                    "query": "source:ossec-security service:syslog @rule_id:(2501 OR 5301) $Level $Souce_IP $Group_List $Host_Name"
```

Comment:
```suggestion
                                    "query": "source:ossec-security service:syslog @rule_id:(2501 OR 5301) $Level $Source_IP $Group_List $Host_Name"
```

Reply:
This misspelling shows up elsewhere as well, so I recommend searching for it and updating the instances

Reply:
Updated as per suggestion